tests/integration_tests/test_mlflow.py

```diff
@@ -425,40 +423,28 @@ def _objective_func(trial: optuna.trial.Trial) -> float:
     assert tracked_objective.__doc__ == _objective_func.__doc__
 
 
-def test_initialize_experiment(tmpdir: py.path.local) -> None:
-    tracking_file_name = "file:{}".format(tmpdir)
-    metric_name = "my_metric_name"
-    study_name = "my_study"
-
-    mlflc = MLflowCallback(tracking_uri=tracking_file_name, metric_name=metric_name)
-    study = optuna.create_study(study_name=study_name)
-
-    mlflc._initialize_experiment(study)
-
-    mlfl_client = MlflowClient(tracking_file_name)
-    experiments = mlfl_client.list_experiments()
-    assert len(experiments) == 1
-
-    experiment = experiments[0]
-    assert experiment.name == study_name
```

Comment:
Ditto.

Reply:
`test_study_name` covers this already, so I removed this duplication.

Reply:
I think this test function aims to verify `mlflc._initialize_experiment(study)`; if we remove `mlflc._initialize_experiment` in this test, we need to rename the test function name or remove this test. So we might need the original test case. 

I suppose `test_log_metric`, `test_log_metric_none`, and `test_log_params` are also same; they aim to verify a primate method of the callaback.

Reply:
I think the same can be archived by testing only using public APIs, regardless of callback implementation details. We test for specific callback behavior (is experiment created with correct name?) which in this case is covered by another test. The test will stay the same even if `mlflc._initialize_experiment` is removed. Same goes for `log_*`

Reply:
Sounds reasonable. Let me resolve this conversation.